test/common/halide_test_dirs.h

```diff
@@ -0,0 +1,69 @@
+#ifndef HALIDE_TEST_DIRS_H
+#define HALIDE_TEST_DIRS_H
+
+// This file may be used by AOT tests, so it deliberately does not
+// include Halide.h
+
+#include <string.h>
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <stdio.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#endif
+
+namespace Halide {
+namespace Internal {
```

Comment:
I don't think it's necessary for this to be in the Internal namespace, given that it's not in Halide.h